tekton/pipeline_run_test.go

```diff
@@ -247,4 +247,59 @@ var _ = Describe("PipelineRun", func() {
 			Expect(releasePipelineRun.Spec.Params).Should(ContainElement(HaveField("Value.StringVal", Equal(string(jsonSpec)))))
 		})
 	})
+
+	Context("When PipelineAsCode metadata is present in the Release", func() {
+		//
+		It("copies Annotations & Labels matching the PAC prefix", func() {
+			release.ObjectMeta.Annotations[pipelinesAsCodeLabelPrefix+"/meow"] = "cat"
+			release.ObjectMeta.Labels = map[string]string{}
+			release.ObjectMeta.Labels[pipelinesAsCodeLabelPrefix+"/bark"] = "dog"
+
+			releasePipelineRun.WithPipelinesAsCodeMeta(release)
+
+			Expect(len(releasePipelineRun.ObjectMeta.Annotations)).To(Equal(1))
+			Expect(len(releasePipelineRun.ObjectMeta.Labels)).To(Equal(1))
+			Expect(releasePipelineRun.ObjectMeta.Annotations[pipelinesAsCodeLabelPrefix+"/meow"]).To(Equal("cat"))
+			Expect(releasePipelineRun.ObjectMeta.Labels[pipelinesAsCodeLabelPrefix+"/bark"]).To(Equal("dog"))
```

Comment:
You think you can use `releasePipelineRun.GetLabels()["key"]`. I don't know if GetAnnotations() would work as well.

Reply:
:smiling_face_with_tear: Regrettably that test was removed during a big refactor. 
:smile: The good news is the tests moved to a new location.
PTAL into `metadata/filter_test.go`